django_hosts/middleware.py

```diff
@@ -15,7 +15,7 @@ class HostsBaseMiddleware(MiddlewareMixin):
     toolbar_middleware = 'debug_toolbar.middleware.DebugToolbarMiddleware'
 
     def __init__(self, get_response=None):
```

Comment:
We can remove the default value now that Django's `__init__()` method requires a non-null value 🙂
```suggestion
    def __init__(self, get_response):
```

Reply:
I forgot that Django < 4.0 still supports this default argument value, so nvm 😅 I instead added a deprecation warning comment 🙂